.github/workflows/ci.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
     - main
+    tags:
+    - v**
```

Comment:
Shouldn't matching a semantic version be more favorable? Something like `v[0-9]+.[0-9]+.[0-9]+**`, which matches for example  `v1.12.199` or `v56.1.0-alpha1+x86_64`?

Reply:
I just build the pattern according to the documentation: [Filter Pattern Cheat Sheet](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet)

Reply:
I don't see a problem switching from my pragmatic approach to the one you proposed. Seems to be more fitting.
👍🏻